Compression/QuantConnect.Compression.csproj

```diff
@@ -33,6 +33,7 @@
     <WarningLevel>4</WarningLevel>
     <LangVersion>6</LangVersion>
     <CodeAnalysisRuleSet>..\QuantConnect.ruleset</CodeAnalysisRuleSet>
+    <DocumentationFile>bin\Debug\QuantConnect.Compression.xml</DocumentationFile>
```

Comment:
Was this one intentional? Looks like the others are limited to release builds.

Reply:
Yes, I thought we might want to have XML docs for this one as well.